### PR TITLE
Create 06-code-of-conduct.md

### PR DESCRIPTION
**Pull request by Balianos**

### CODE REVIEW

**Review by M2613 — CHANGES_REQUESTED** [review]

File: docs/general/06-code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at:
+info@mystenlabs.com
```
[the remark this answers is not in the document]
Comment (Balianos) [reply docs/general/06-code-of-conduct.md:70]: Un oubli, dsl

**Review by Balianos — COMMENTED** [review]
Updated

File: docs/general/06-code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at:
+support@ternoa.com
```
[the remark this answers is not in the document]
Comment (Balianos) [reply docs/general/06-code-of-conduct.md]: @M2613 Done

**Comment by Balianos** [comment]
@M2613 I think we're good and we can merge this =)

**Review by M2613 — APPROVED** [review]

**Pull request merged**